docs/reference/readme.md

```diff
@@ -102,22 +102,26 @@ var StellarSdk = require('stellar-sdk')
 StellarSdk.Network.useTestNetwork();
 var server = new StellarSdk.Server('https://horizon-testnet.stellar.org');
 
-var transaction = new StellarSdk.TransactionBuilder(account)
-        // this operation funds the new account with XLM
-        .addOperation(StellarSdk.Operation.payment({
-            destination: "GASOCNHNNLYFNMDJYQ3XFMI7BYHIOCFW3GJEOWRPEGK2TDPGTG2E5EDW",
-            asset: StellarSdk.Asset.native(),
-            amount: "20000000"
-        }))
-        .build();
-
-transaction.sign(StellarSdk.Keypair.fromSecret(secretString)); // sign the transaction
-
-server.submitTransaction(transaction)
-    .then(function (transactionResult) {
-        console.log(transactionResult);
-    })
-    .catch(function (err) {
-        console.error(err);
-    });
+server
+  .loadAccount(publicKey)
+  .then(function(account){
+  		var transaction = new StellarSdk.TransactionBuilder(account)
+  				// this operation funds the new account with XLM
+  				.addOperation(StellarSdk.Operation.payment({
+  					destination: "GASOCNHNNLYFNMDJYQ3XFMI7BYHIOCFW3GJEOWRPEGK2TDPGTG2E5EDW",
+  					asset: StellarSdk.Asset.native(),
+  					amount: "20000000"
+  				}))
+  				.build();
```

Comment:
I'm wondering if such a big indent will render correctly at https://www.stellar.org/developers/js-stellar-sdk/reference/index.html.

Reply:
if it does, we'll fix it.